README.md

```diff
@@ -180,3 +180,4 @@ category: "design"
 ├── README.md
 └── science.md
 ```
+          &copy; 2017
```

Comment:
The text does not comply to the set style.

*Origin: MarkdownBear, Section: `markdown`.*

The issue can be fixed by applying the following patch:

```diff
--- a/README.md
+++ b/README.md
@@ -180,4 +180,7 @@
 ├── README.md
 └── science.md
 ```
-          &copy; 2017
+
+```
+      &copy; 2017
+```
```